### PR TITLE
feature/grants-3940

### PR DESCRIPTION
**Pull request by Hozan94**

### CODE REVIEW

File: app/helpers/handlers.js [hunk 1]
```diff
@@ -82,6 +84,11 @@ const insertYarValue = (field, url, request) => {
     field = field.replace('[[_article_]]', replacement)
   }
 
+  if (url === 'veranda-features') {
```
Comment (Hozan94) [inline app/helpers/handlers.js:87]: There is another method which achieves the same results here. Check the very first commit to see which is more optimal.

Comment (zavies97) [reply app/helpers/handlers.js:87]: As long as everything works across the app when you test it locally (via docker), then its all fine :)

Just conscious that writing code specifically for one page isn't very generalised :P

Comment (zavies97) [reply app/helpers/handlers.js:87]: I think the first commit version was better as it fit into the existing structure, as opposed to creating a new and specific replacement for one page only

Comment (Hozan94) [reply app/helpers/handlers.js:87]: okay then, will use the first commit version 😃

**Review by zavies97 — APPROVED** [review]

**Review by zavies97 — CHANGES_REQUESTED** [review]
Just saw that **Farming Transformation Fund** is actually **Farming Investment Fund** for accessibility.njk

Looks like there may also be other changes needed there as per grants-3944, so worth a double check to make sure all new text is added/updated correctly :)

**Comment by Hozan94** [comment]
> Just saw that **Farming Transformation Fund** is actually **Farming Investment Fund** for accessibility.njk
> 
> Looks like there may also be other changes needed there as per grants-3944, so worth a double check to make sure all new text is added/updated correctly :)

**Comment by Hozan94** [comment]
> Just saw that **Farming Transformation Fund** is actually **Farming Investment Fund** for accessibility.njk
> 
> Looks like there may also be other changes needed there as per grants-3944, so worth a double check to make sure all new text is added/updated correctly :)

Thank you for reminding me, totally missed it.

Also this part is actually already in place so no change is needed
```
This service is partially compliant with the Web Content Accessibility Guidelines version 2.2 AA standard, due to the non-compliances listed.

We intend to fix these problems by as soon as possible.
```

**Review by zavies97 — APPROVED** [review]

**Pull request merged**